Source/BugsnagNotifier.m

```diff
@@ -411,6 +411,11 @@ - (void)start {
 
     [self.sessionTracker startNewSessionIfAutoCaptureEnabled];
 
+    [self addBreadcrumbWithBlock:^(BugsnagBreadcrumb *_Nonnull breadcrumb) {
+         breadcrumb.type = BSGBreadcrumbTypeState;
+         breadcrumb.name = @"Bugsnag loaded";
```

Comment:
Should this be a constant? Breadcrumb names seem to have a fair amount of indirection already.  Is there any value in concentrating possible string values (for automatically captured breadcrumbs) in a central location?

Reply:
I thought something similar, though I think automatic breadcrumbs overall needs a better home than this class and a place for all of the constants and conversions. In the meantime, I've moved the name into a constant and added a backlog refactor task to figure out a proper place for these.